services/core/PlatformDriverAgent/platform_driver/interfaces/udd_dnp3/dnp3-driver.md

```diff
@@ -0,0 +1,75 @@
+
```

Comment:
Should this PR also update the readthedocs for DNP3? We currently have https://volttron.readthedocs.io/en/develop/agent-framework/driver-framework/dnp3-driver/dnp3-driver.html. Does this PR automagically update the readthedocs for DNP3?

Reply:
> Should this PR also update the readthedocs for DNP3? We currently have https://volttron.readthedocs.io/en/develop/agent-framework/driver-framework/dnp3-driver/dnp3-driver.html. Does this PR automagically update the readthedocs for DNP3?

Does this PR automagically update the readthedocs for DNP3?
Short answer: no it does not.

Long answer:
It depends on how we deal with the old dnp3-driver at services/core/PlatformDriverAgent/platform_driver/interfaces/dnp3.py, since the current rtd is associated with it.

Reply:
Is your PR a replacement for the old DNP3 driver? I recall having trouble using the old dnp3 driver. If this new feature can do the same job as the old DNP3 driver, then I'd be more inclined to replace the old dnp3 driver with this one. If so, then we should also update the rtd.

Reply:
Noted. The ultimate goal is to supersede the old one but there is some details when creating this PR. e.g., should I put the old driver to deprecate, or should I just delete it? So, for now, I create incremental PR for the purpose.

Reply:
I'm for deprecating the old DNP3 driver. We can bring it up during the Wed working meeting with the rest of the team.

Reply:
Updated: with commit [deprecated old dnp3-driver code and updated dnp3-driver rtd](https://github.com/VOLTTRON/volttron/pull/3055/commits/702a2ea913365a2fd526a00637ec2e523669dfad)

Deprecated old dnp3-driver code
Updated documentation
See discussion at: https://github.com/VOLTTRON/volttron/pull/3055#issuecomment-1301579685